README.md

```diff
@@ -34,241 +30,10 @@ If you already cloned and just need to get the submodules, you can run:
 ```sh
 git submodule update --init --recursive
 ```
+## Official Documentation
 
-### Dependencies
-
-Make sure you have installed the packages listed in `environment.yml`.
-When listed, specific particular package versions are required.
-If you use `conda` (recommended, [instructions for installing `miniconda` here](https://conda.io/miniconda.html)), you can create an environment from this package with the following command:
-
-```
-conda env create -f environment.yml
-```
-
-To activate the environment run ``source activate jiant``, and to deactivate run ``source deactivate``
-
-Some requirements may only be needed for specific configurations. If you have trouble installing a specific dependency and suspect that it isn't needed for your use case, create an issue or a pull request, and we'll help you get by without it.
-
-You will also need to install dependencies for nltk if you do not already have them:
-```
-python -m nltk.downloader -d /usr/share/nltk_data perluniprops nonbreaking_prefixes punkt
-```
-
-
-### Downloading data
-
-The repo contains a convenience python script for downloading all [GLUE](https://gluebenchmark.com/) data and standard splits.
-
-```
-python scripts/download_glue_data.py --data_dir data --tasks all
-```
-
-We also make use of many other data sources, including:
```

Comment:
This should stick around in some kind of documentation.

Reply:
Yes, it is present in the Getting Started tutorial as well as here: https://jiant.info/documentation#/?id=data-sources